CHANGELOG.md

```diff
@@ -8,6 +8,25 @@
 
 ### Bug fixes
 
+## 1.9.4 (5 Apr 2019)
+
+### New features
```

Comment:
```suggestion
### Breaking Changes 

- `GraphQL::Schema::Resolver::LoadApplicationObjectFailedError` was renamed to `GraphQL::LoadApplicationObjectFailedError`. (This will only break if you're referencing the class by name and running Ruby 2.5+) #2080 

### New features

```